core/feed/annotator/circulation.py

```diff
@@ -927,7 +927,7 @@ def annotate_work_entry(
                 )
             )
 
-        if Analytics.is_configured(self.library):
+        if container_instance().analytics.analytics().is_configured():
```

Comment:
I'd like to see this passed as configuration into the `__init__` for `LibraryAnnotator` rather then accessing the global object like this.